### PR TITLE
Km update Paired-Tag, Optimus, GDC, multi-snSS2 docs

### PR DESCRIPTION
**Pull request by kayleemathews**
### Description

In this PR, I'm making updates to the Paired-Tag, Optimus, GDC, and multi-snSS2 Overview docs.

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

File: website/docs/Pipelines/Genomic_Data_Commons_Whole_Genome_Somatic/README.md [hunk 1]
```diff
@@ -29,8 +29,8 @@ For the latest workflow version and release notes, please see the [changelog](ht
 
 ### Software version requirements
 
-* GATK 4.0.7
-* Picard 2.18.11 (Custom Docker is used to run software on Cromwell 52)
+* GATK 4.5.0.0
```
[the remark this answers is not in the document]
Comment (kayleemathews) [reply website/docs/Pipelines/Genomic_Data_Commons_Whole_Genome_Somatic/README.md:32]: Oh right! I will wait to merge until that actually gets updated.

[the remark this answers is not in the document]
Comment (kayleemathews) [reply website/docs/Pipelines/Genomic_Data_Commons_Whole_Genome_Somatic/README.md:32]: Thanks for catching that!

**Review by ekiernan — APPROVED** [review]
Thanks, Kaylee!

File: website/docs/Pipelines/Genomic_Data_Commons_Whole_Genome_Somatic/README.md [hunk 2]
```diff
@@ -29,8 +29,8 @@ For the latest workflow version and release notes, please see the [changelog](ht
...
+* Picard 2.26.10 (Custom Docker is used to run software on Cromwell 52)
```
[the remark this answers is not in the document]
Comment (kayleemathews) [reply website/docs/Pipelines/Genomic_Data_Commons_Whole_Genome_Somatic/README.md:33]: @jessicaway Based on [this commit](https://github.com/broadinstitute/warp/commit/cb7ad710065cca00afb55bf59de4066dfbe0f127) it seems like it's probably not true anymore?

File: website/docs/Pipelines/PairedTag_Pipeline/README.md [hunk 3]
```diff
@@ -105,7 +106,7 @@ ATAC ([WDL](https://github.com/broadinstitute/warp/blob/develop/pipelines/skylab
 |--- | --- | --- | 
 | pairedtag_pipeline_version_out | N.A. | String describing the version of the Paired-Tag pipeline used. |
 | bam_aligned_output_atac | `<input_id>_atac.bam` | BAM file containing aligned reads from ATAC workflow; contains sample and cell barcodes stored in the BB tag if `preindex` is “true”. |
-| fragment_file_atac | `<input_id>_atac.fragments.tsv` or if preindexing = true, `<input_id>_atac.fragments.BB.tsv | TSV file containing fragment start and stop coordinates per barcode. The columns are "Chromosome", "Start", "Stop", "Barcode", and "Number of reads". | 
+| fragment_file_atac | `<input_id>_atac.fragments.tsv` or if preindexing = true, `<input_id>_atac.fragments.BB.tsv` | TSV file containing fragment start and stop coordinates per barcode. The columns are "Chromosome", "Start", "Stop", "Barcode", and "Number of reads". |
```
Comment (ekiernan) [inline website/docs/Pipelines/PairedTag_Pipeline/README.md:109]: ```| fragment_file_atac | `<input_id>_atac.fragments.tsv` or if preindexing = true, `<input_id>_atac.fragments.BB.tsv` | TSV file containing fragment start and stop coordinates per barcode. The columns are "Chromosome", "Start", "Stop", "Barcode", and "Number of reads". When preindexing is used, additional columns include "Sample Barcode", "Cell Barcode", and "Duplicates" (which indicates if a cell barcode matches more than one sample barcode).  |

**Pull request merged**